backend/server.js

```diff
@@ -0,0 +1,224 @@
+const express = require("express");
+const cors = require("cors");
+const multer = require("multer");
+
+const upload = multer();
+
+const cookieParser = require("cookie-parser");
+const app = express();
+
+const MongoClient = require("mongodb").MongoClient;
+
+const passwordCheck = require(__dirname + "/utilities/passwordCheck.js");
```

Comment:
You can just `require('./utilities/passwordCheck')`. You don't need __dirname, nor the `.js` extension =)

If you ran into some funky issues doing it that way, lemme know and I'll take a closer look